### PR TITLE
When sending email to reviewers, send a copy to owner

### PR DESCRIPTION
**Pull request by chrisvire**
* Include who the message was sent to and a copy of the message.

### CODE REVIEW

File: source/SIL.AppBuilder.Portal.Backend.Tests/source/locales/en-us.json [hunk 1]
```diff
@@ -1,42 +1,43 @@
...
+        "productCreationFailedOwner": "<p>Exceeded retries attempting to create product: {{productName}} project {{projectName}}.</p>The organization administrator has been notified of this issue.",
+        "productCreationFailedAdmin": "<p>Exceeded retries attempting to create product: {{productName}} project {{projectName}}.</p>",
+        "productRecordNotFound": "<p>Product Creation Failed: Unable to find product record with id: {{productId}}</p>",
+        "releaseFailedUnableToConnect": "<p>Failed to create release for product: {{productName}} project: {{projectName}}.</p><p>Unable to connect to build engine.</p>",
+        "releaseFailedUnableToCreate": "<p>Failed to create release for product: {{productName}} project: {{projectName}}.</p><p>Retries exceeded for attempt to create release on build machine.</p>",
+        "releaseCompletedSuccessfully": "<p>Release for product: {{productName}} project: {{projectName}} completed successfully</p>",
+        "reviewOwnerPrefix": "<p>{{ownerName}},</p><p>The following message was sent to the these reviewers:<br>{{reviewerNames}}<br></p><hr/>",
```
Comment (davidmoore1) [inline source/SIL.AppBuilder.Portal.Backend.Tests/source/locales/en-us.json:32]: Same as above

**Review by davidmoore1 — APPROVED** [review]
Approve with the one minor comment that you can take or leave

File: source/OptimaJet.DWKit.StarterApplication/source/locales/en-us.json [hunk 2]
```diff
@@ -1,41 +1,42 @@
...
+      "releaseFailedUnableToCreate": "<p>Failed to create release for product: {{productName}} project: {{projectName}}.</p><p>Retries exceeded for attempt to create release on build machine.</p>",
+      "releaseCompletedSuccessfully": "<p>Release for product: {{productName}} project: {{projectName}} completed successfully</p>",
+      "releaseFailedOwner": "<p>Publish for product: {{productName}} project {{projectName}} failed.</p><p>Project URL: {{projectUrl}}</p><p> The organization administrator has been notified of this issue.</p>",
+      "releaseFailedAdmin": "<p>Publish for product: {{productName}} project {{projectName}} failed.</p><p>Project URL: {{projectUrl}}</p><p>Diagnostic information for Super Admins:</p><ul><li>BuildEngineUrl: {{buildEngineUrl}}</li><li>ProjectId: {{projectId}} </li><li>JobId: {{jobId}} </li><li> BuildId: {{buildId}}</li><li>PublishId: {{publishId}} </li></ul>",
+      "releaseProductRecordNotFound": "<p>Release Failed: Unable to find product record with id: {{productId}}</p>",
+      "reviewOwnerPrefix": "<p>{{ownerName}},</p><p>The following message was sent to the these reviewers:<br>{{reviewerNames}}<br></p><hr/>",
```
[the remark this answers is not in the document]
Comment (chrisvire) [reply source/OptimaJet.DWKit.StarterApplication/source/locales/en-us.json:35]: I wondered why the `<br>` were not `<br/>`.

[the remark this answers is not in the document]
Comment (chrisvire) [reply source/OptimaJet.DWKit.StarterApplication/source/locales/en-us.json:35]: Switching to `<hr>`

File: source/SIL.AppBuilder.Portal.Backend.Tests/source/locales/en-us.json (hunk 1 above)
Comment (chrisvire) [reply source/SIL.AppBuilder.Portal.Backend.Tests/source/locales/en-us.json:32]: Switching to `<hr>`

**Pull request merged**